js/app.js

```diff
@@ -12,4 +15,40 @@ export default {
   getSocketUrl() {
     return 'ws://localhost:8080/ws';
   },
+
+  followUnfollow(guid, type = 'follow') {
```

Comment:
It's not a great name, I admit, I couldn't think of something better to describe "does things related to following, but isn't called follow.js".